cabot/metricsapp/api/grafana_elastic.py

```diff
@@ -192,3 +197,29 @@ def get_es_status_check_fields(dashboard_info, panel_info, series_list):
         fields['queries'].append(query)
 
     return fields
+
+
+def adjust_time_range(queries, time_range):
```

Comment:
Change the query if the user specifies a smaller time range so we don't query for data we don't need